web/public/locales/de.json

```diff
@@ -170,13 +170,15 @@
   "misc": {
     "maintitle": "CO₂-Emissionen des Stromverbrauchs in Echtzeit",
     "oops": "Ups! Wir haben Probleme den Server zu erreichen. Wir versuchen es in wenigen Sekunden noch einmal.",
-    "newversion": "Eine neuere Version ist verfügbar! Klick <a onClick=\"location.reload(true);\">hier</a>, um sie zu laden.",
+    "newversion": "Eine neuere Version ist verfügbar!",
     "retrynow": "Erneut versuchen",
     "database-ad": "Auf der Suche nach historischen Daten? <a href=\"https://electricitymaps.com?utm_source=app.electricitymaps.com&utm_medium=referral\" target=\"_blank\">Sieh dir unsere Datenbank an!</a>",
     "api-ad": "Benötigst du Live-Daten für deine Apps oder Geräte? <a href=\"https://electricitymaps.com?utm_source=app.electricitymaps.com&utm_medium=referral\" target=\"_blank\">Probiere unsere API!</a>",
     "legend": "Legende",
     "faq": "Häufig gestellte Fragen",
-    "webgl-not-supported": "Diese Karte kann nicht angezeigt werden, weil dieser Webbrowser kein WebGL unterstützt."
+    "webgl-not-supported": "Diese Karte kann nicht angezeigt werden, weil dieser Webbrowser kein WebGL unterstützt.",
+    "dismiss": "Zurückweisen",
```

Comment:
I'm not native in German but as far as I can tell, "zurückweisen" also has a stronger sense of "reject, refuse, dispute" that isn't quite right here

Reply:
This has been updated to "Schließen" which more closely resembles "close" as the usage is to close the popup.